Doc/using/unix.rst

```diff
@@ -72,15 +72,15 @@ latest release's source or just grab a fresh `clone
 <https://devguide.python.org/setup/#getting-the-source-code>`_.  (If you want
 to contribute patches, you will need a clone.)
 
-The build process consists in the usual ::
+The build process consists in the usual commands. ::
```

Comment:
We lost the `s/consists in/consists of/` fix.  Also, this should not end with a period:

```suggestion
The build process consists of the usual commands::
```

Reply:
yep, I don't know why (for the lost)